src/main/java/gregtech/integration/forestry/recipes/CombRecipes.java

```diff
@@ -0,0 +1,493 @@
+            addCentrifugeToMaterial(GTCombType.MAGNESIUM, new Material[]{Materials.Magnesium, Materials.Magnesite}, new int[]{100 * 100, 80 * 100}, new int[]{}, Voltage.ULV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.CHROME, new Material[]{Materials.Chrome, Materials.Ruby, Materials.Chromite, Materials.Redstone, Materials.Neodymium, Materials.Bastnasite}, new int[]{50 * 100, 100 * 100, 50 * 100, 100 * 100, 80 * 100, 80 * 100}, new int[]{}, Voltage.HV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.TUNGSTEN, new Material[]{Materials.Tungsten, Materials.Tungstate, Materials.Scheelite, Materials.Lithium}, new int[]{50 * 100, 80 * 100, 75 * 100, 75 * 100}, new int[]{}, Voltage.IV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.PLATINUM, new Material[]{Materials.Platinum, Materials.Cooperite, Materials.Palladium}, new int[]{40 * 100, 40 * 100, 40 * 100}, new int[]{}, Voltage.HV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.MOLYBDENUM, new Material[]{Materials.Molybdenum, Materials.Molybdenite, Materials.Powellite, Materials.Wulfenite}, new int[]{100 * 100, 80 * 100, 75 * 100}, new int[]{}, Voltage.ULV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.IRIDIUM, new Material[]{Materials.Iridium, Materials.Osmium}, new int[]{20 * 100, 15 * 100}, new int[]{}, Voltage.IV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.OSMIUM, new Material[]{Materials.Osmium, Materials.Iridium}, new int[]{25 * 100, 15 * 100}, new int[]{}, Voltage.IV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.LITHIUM, new Material[]{Materials.Lithium, Materials.Aluminium}, new int[]{85 * 100, 75 * 100}, new int[]{}, Voltage.MV, ItemStack.EMPTY, 30 * 100);
+            addCentrifugeToMaterial(GTCombType.SALT, new Material[]{Materials.Salt, Materials.RockSalt, Materials.Saltpeter}, new int[]{100 * 100, 75 * 100, 65 * 100}, new int[]{9, 4, 4}, Voltage.MV, 160, ItemStack.EMPTY, 50 * 100);
+            addCentrifugeToMaterial(GTCombType.ELECTROTINE, new Material[]{Materials.Electrotine, Materials.Electrum, Materials.Redstone}, new int[]{80, 75, 65}, new int[]{}, Voltage.MV, ItemStack.EMPTY, 30 * 100);
+        }
+
+        // Special Indium Recipe
+        RecipeMaps.CHEMICAL_RECIPES.recipeBuilder()
+                .input(OrePrefix.dust, Materials.Aluminium, 4)
+                .inputs(ForestryUtil.getCombStack(GTCombType.INDIUM))
+                .fluidInputs(Materials.IndiumConcentrate.getFluid(1000))
+                .output(OrePrefix.dustSmall, Materials.Indium, 2)
+                .output(OrePrefix.dust, Materials.AluminiumSulfite, 4)
+                .fluidOutputs(Materials.LeadZincSolution.getFluid(1000))
+                .duration(50).EUt(600).buildAndRegister();
```

Comment:
This recipe causes a conflict with the regular indium recipe. The regular one will need an config circuit to rectify this.